### PR TITLE
Completed javascript-timers. - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Kapture 2023-06-05 at 13 58 17

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+let count = 4;
+
+function updateCountdown() {
+  if (count === 0) {
+    $countdownDisplay.textContent = `~Earth Beeeelooowww~`;
+    clearInterval();
```
Comment (shawnkost) [inline javascript-timers/interval.js:7]: `clearInterval` should be getting called with 1 argument

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Jesus!

File: javascript-timers/interval.js (hunk 1 above)
Comment (j-alejandro-araujo) [reply javascript-timers/interval.js]: Screenshot 2023-06-05 at 2 12 24 PM

Adjusted!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**